LICENSE.md

```diff
@@ -0,0 +1,27 @@
+Copyright (c) 2020, andreakarasho
+All rights reserved.
+
+Redistribution and use in source and binary forms, with or without
+modification, are permitted provided that the following conditions are met:
+1. Redistributions of source code must retain the above copyright
+   notice, this list of conditions and the following disclaimer.
+2. Redistributions in binary form must reproduce the above copyright
+   notice, this list of conditions and the following disclaimer in the
+   documentation and/or other materials provided with the distribution.
+3. All advertising materials mentioning features or use of this software
+   must display the following acknowledgement:
+   This product includes software developed by the copyright holder.
```

Comment:
This line should be:

```
This product includes software developed by andreakarasho.
```

Section 3 indicates that this acknowledgment must appear verbatim. In its current form, advertising materials would have to include literal text "This product includes software developed by the copyright holder." which carries no meaning in that context.